packages/core/src/modules/attendance/attendee-service.ts

```diff
@@ -28,12 +28,12 @@ export interface AttendeeService {
   registerForEvent(userId: string, attendanceId: string, attendancePoolId: string): Promise<Attendee>
   adminRegisterForEvent(userId: string, attendanceId: string, attendancePoolId: string): Promise<Attendee>
   deregisterForEvent(userId: string, attendanceId: string): Promise<void>
-  adminDeregisterForEvent(id: AttendeeId): Promise<void>
+  adminDeregisterForEvent(id: AttendeeId, reserveNext?: boolean): Promise<void>
```

Comment:
Suggest not making service call parameters optional, especially booleans. It is more explicit this way